README.md

```diff
@@ -622,12 +622,29 @@ Note: untrack doesn't get you out of the `notifying` state, which is maintained
 
 **Q**: What are Signals offering that `Proxy` doesn't currently handle?
 
-**A**: Proxies must wrap some object. They cannot be used to intercept property access/assignment to primitive values such as numbers, strings, or symbols.
-See Prior Implementations [tracked-built-ins](https://github.com/tracked-tools/tracked-built-ins/tree/master/addon/src/-private), [discussion](https://github.com/proposal-signals/proposal-signals/issues/101#issuecomment-2029802574). The following is valid for a Signal, but not for a Proxy:
+**A**: Proxies must wrap some object and do not inherently participate in a reactive graph. the reactive graph is built by objects that know about and discover other reactive data during the calculation of a value. Probies also have a usage limitation: they cannot be used to intercept property access/assignment to primitive values such as numbers, strings, or symbols.
```

Comment:
There's a typo 😅 "Probies" should be "Proxies" in:

> Probies also have a usage limitation

Reply:
Thanks!